keyboards/pabile/p18/keymaps/default/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(
+					KC_NO,KC_PMNS, KC_P9, KC_P8, KC_P7, 
+					KC_NO,KC_PPLS,KC_P6, KC_P5, KC_P4, 
```

Comment:
Why the KC_NO's here?

Reply:
There are no physical switches on these first part of the grid. I may have not seen all but i could not seem to find on how to properly address such case.

Reply:
> Why the KC_NO's here?

Hi! I just figured this out and removed KC_NOs. I also updated p18.h. Thanks for checking.